`import/no-unresolved` on NX monorepos starting from version `3.8.x`

Hi,
Starting from version `3.8.x` of `eslint-import-resolver-typescript`, I started getting `import/no-unresolved` on references between projects in my NX monorepo with aliases.

It worked fine until version `3.7.0`, but it stopped working since. I also tested it on the recently released `3.8.6` but it's still the same issue.

There are some related issues starting from `3.8.x`, but this issue wasn't resolved as part of the fixes:
* https://github.com/import-js/eslint-import-resolver-typescript/issues/364
* https://github.com/import-js/eslint-import-resolver-typescript/issues/363
* https://github.com/import-js/eslint-import-resolver-typescript/issues/359

I created a reproduction repo [here](https://github.com/estiller/eslint-import-resolver-issue). It has two libraries, with `my-lib2` referencing `my-lib`.

Reproduction steps:
1. Clone the repository and run `pnpm install`. This will install version `3.7.0`.
2. Run `pnpm lint`. Everything should work fine.
3. Run `pnpm add -D eslint-import-resolver-typescript@3.8.6`.
4. Run `pnpm lint` and you will get a lint error.

```
   ✖  nx run my-lib2:lint
      > eslint .
      
      
      /Users/eran/estiller/eslint-import-resolver-issue/libs/my-lib2/src/lib/my-lib2.ts
        1:23  error  Unable to resolve path to module '@eslint-import-resolver-issue/my-lib'  import/no-unresolved
      
      ✖ 1 problem (1 error, 0 warnings)
```

I'd appreciate any help. Thanks!

--- a/.changeset/new-peas-fry.md
+++ b/.changeset/new-peas-fry.md
@@ -1,0 +1,5 @@
+---
+'eslint-import-resolver-typescript': patch
+---
+
+fix: include mapper with no files and force non-dynamic projects to use absolute paths

--- a/src/index.ts
+++ b/src/index.ts
@@ -430,7 +430,9 @@ function initMappers(options: InternalResolverOptions) {
   // Turn glob patterns into paths
   const projectPaths = [
     ...new Set([
-      ...configPaths.filter(path => !isDynamicPattern(path)),
+      ...configPaths
+        .filter(p => !isDynamicPattern(p))
+        .map(p => path.resolve(process.cwd(), p)),
       ...globSync(
         configPaths.filter(path => isDynamicPattern(path)),
         {
@@ -503,11 +505,6 @@ function initMappers(options: InternalResolverOptions) {
                   })
                 : []),
             ]
-
-      if (files.length === 0) {
-        // eslint-disable-next-line unicorn/no-useless-undefined
-        return undefined
-      }
 
       return {
         path: toNativePathSeparator(tsconfigResult.path),

